monty/exts/info/github_info.py

```diff
@@ -920,8 +933,27 @@ async def on_message_automatic_issue_link(self, message: disnake.Message) -> Non
         if not links:
             return
 
-        expand_one_issue = await self.bot.guild_has_feature(message.guild, ISSUE_EXPAND_FEATURE_NAME)
-        embed = self.format_embed(links)
+        if len(links) == 1 and issues[0].should_be_expanded:
+
+            async def remove_embeds() -> None:
+                if not message.embeds:
+                    try:
+                        await self.bot.wait_for("message_edit", check=lambda b, m: m.id == message.id, timeout=3)
+                    except asyncio.TimeoutError:
+                        pass
+                await message.edit(suppress_embeds=True)
+
+            scheduling.create_task(remove_embeds())
```

Comment:
The codesnippets handler does something similar, could perhaps be refactored into a separate function?

Reply:
Possibly, but not right now.